pages/news.html

```diff
@@ -142,14 +142,14 @@ <h1>NEWS</h1>
             <p><strong>24/07/2020</strong> - Uyuni 2020.07 is now available! Base OS for server and proxy is now Leap 15.2 and it includes PostgreSQL 12 and Salt 3000. Ubuntu 20.04 can now be managed as a client, and much more changes! Check <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details!</p>
             <p><strong>18/06/2020</strong> - Next June 26th, at 16.00 CEST, we will have a new Uyuni Community Hours session. Check all the <a href="https://lists.opensuse.org/uyuni-announce/2020-06/msg00002.html" target="_blank">details</a> and join us!</p>
             <p><strong>15/06/2020</strong> - Uyuni 2020.06 is now available! This release adds support for Oracle Linux and CaaSP cluster management. Check the full <a href="../pages/stable-version.html#releasenotes">release notes</a>!</p>
-            <p><strong>04/06/2020</strong> - We are now moving our chat to <a href="https://gitter.im/uyuni-project" target="_blank">Gitter</a>. The old IRC channel is now frozen and only exists to redirect people there. See the <a href="https://lists.opensuse.org/uyuni-announce/2020-06/msg00000.html" target="_blank">official announcement</a>.</p>
+            <p><strong>04/06/2020</strong> - We are now moving our chat to <a href="../pages/contact.html#gitter">Gitter</a>. The old IRC channel is now frozen. See the <a href="https://lists.opensuse.org/uyuni-announce/2020-06/msg00000.html" target="_blank">official announcement</a>.</p>
```

Comment:
Strictly speaking, the old IRC channel does not even exist anymore. But let's leave this here, for historic reference :-)

Reply:
adjusted the link. the  old is broken